### PR TITLE
Fix:PRO-6067:Fix for Sidebar and Quick Links

### PR DESCRIPTION
**Pull request by austinjb32**
## What does this change do?

-  Alignment Fix for Sidebar and Quick Links
- Making contents in Sidebar and Quick Links same

### Before Change
image

### After Change

**Sidebar**
image image

**Quick Links**
image image

**Contents**
We made sure the contents of the sidebar are the same as of quick links
image

### CODE REVIEW

File: app/assets/stylesheets/cm_admin/base/sidebar.scss [hunk 1]
```diff
@@ -47,14 +47,28 @@
     height: calc(100vh - 180px);
     padding: 16px 0;
     overflow-y: auto;
+
     .menu-item {
       @include font($size: $t4-text, $color: $ink-lightest-clr);
       padding: 8px 24px;
       transition: all 0.2s linear;
       cursor: pointer;
+      display: flex;
```
Comment (barath46) [inline app/assets/stylesheets/cm_admin/base/sidebar.scss:56]: Use `flex mixins`

File: app/assets/stylesheets/cm_admin/base/sidebar.scss [hunk 2]
```diff
@@ -47,14 +47,28 @@
...
+      align-items: center;
       .menu-icon {
+        display: flex;
```
Comment (barath46) [inline app/assets/stylesheets/cm_admin/base/sidebar.scss:60]: Use `flex mixins`

**Review by barath46 — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/cm_admin/base/sidebar.scss (hunk 1 above)
Comment (austinjb32) [reply app/assets/stylesheets/cm_admin/base/sidebar.scss]: @barath46  the changes are made

File: app/assets/stylesheets/cm_admin/base/sidebar.scss (hunk 2 above)
Comment (austinjb32) [reply app/assets/stylesheets/cm_admin/base/sidebar.scss]: @barath46  the changes are done

**Review by barath46 — APPROVED** [review]

**Review by mikevic — APPROVED** [review]

**Review by Mahaveer1141 — APPROVED** [review]

**Pull request merged**